--- /usr/share/php/File/Archive/Writer/Files.php	2005-08-23 13:37:13.000000000 +0100
+++ Files.php	2005-08-30 10:20:48.000000000 +0100
@@ -237,12 +237,12 @@
             if (isset($this->stat[2])) {
                 chmod($this->filename, $this->stat[2]);
             }
-            if (isset($this->stat[5])) {
-                chgrp($this->filename, $this->stat[5]);
-            }
-            if (isset($this->stat[4])) {
-                chown($this->filename, $this->stat[4]);
-            }
+#            if (isset($this->stat[5])) {
+#                chgrp($this->filename, $this->stat[5]);
+#            }
+#            if (isset($this->stat[4])) {
+#               chown($this->filename, $this->stat[4]);
+#            }
         }
     }
 }
